pkg/core/blockchain.go

```diff
@@ -1757,3 +1811,41 @@ func (bc *Blockchain) newInteropContext(trigger trigger.Type, d dao.DAO, block *
 func (bc *Blockchain) P2PSigExtensionsEnabled() bool {
 	return bc.config.P2PSigExtensions
 }
+
+// RegisterPostBlock appends provided function to the list of functions which should be run after new block
+// is stored.
+func (bc *Blockchain) RegisterPostBlock(f func(blockchainer.Blockchainer, *mempool.Pool, *block.Block)) {
+	bc.postBlock = append(bc.postBlock, f)
+}
+
+// RunWithCurrentState allows to call f under the Blockchain lock.
+func (bc *Blockchain) RunWithCurrentState(f func() error) error {
```

Comment:
I don't see it being used now, seems like `PoolTxWithData` completely replaced it.